csuntechlab/iSTART
# STR-191 -- FE: Go through 2nd revisions of Alcohol module

# Description
Revised the Alcohol Module content based off the 2nd revisions requested from the client in the last demo.
  
# Testing
1. Login through the Module route.
2. Make sure all the 2nd revision requested changes are implemented. (*will talk to client about some content that hasn't been implemented due to question/concerns*)
